src/Wrap.js

```diff
@@ -68,5 +68,15 @@ const Wrap = (props: WrapProps): React.Element<*> => {
     </svg>
   )
 }
+Wrap.defaultProps = {
```

Comment:
There is a reason why I did like that, please take a look at this PR:
https://github.com/danilowoz/react-content-loader/pull/72

Reply:
👍, got it. Forgive me for not looking at the previous PR。

Reply:
No problem :)